app/controllers/katello/api/rhsm/candlepin_dynflow_proxy_controller.rb

```diff
@@ -31,19 +29,6 @@ def upload_profiles
       render :json => Resources::Candlepin::Consumer.get(@host.subscription_facet.uuid)
     end
 
-    api :PUT, "/systems/:id/deb_package_profile", N_("Update installed deb packages")
-    param :deb_package_profile, Hash, :required => true do
-      param :deb_packages, Array, :required => true do
-        param :name, String, :required => true
-        param :architecture, String, :required => true
-        param :version, String, :required => true
-      end
-    end
-    param :id, String, :desc => N_("UUID of the system"), :required => true
-    def deb_package_profile
-      upload_profiles
-    end
-
```

Comment:
@jturel since you already reviewed this and it looks like you messed around with this deb_package_profile api previously, can you give some backstory on what it was used for and why its going away?

Reply:
ATIX added this in support of deb package actions before subscription-manager could upload the deb package profile (they had a separate script to upload the package data). Subman has been able to do that for a while now, so I think it's safe to remove this API.